src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,42 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" {})
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (apply + (map count-map ["Queued" "Launching"]))
```

Comment:
The counts I'm showing here are the same as those defined in Rawls (see https://github.com/broadinstitute/rawls/blob/e6be332405bfa36c35f935d7ea613e46a11feca8/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala#L294) I'm taking the Rawls version as the correct counts, but that may be incorrect. I ended up not using `queue-status-remap` because it seems to be doing things differently than what's shown in `ExecutionModel`